src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-restore-modal/rbd-trash-restore-modal.component.html

```diff
@@ -0,0 +1,53 @@
+<cd-modal>
+  <ng-container i18n
+                class="modal-title">Restore Image</ng-container>
+
+  <ng-container class="modal-content">
+    <form name="restoreForm"
+          class="form"
+          #formDir="ngForm"
+          [formGroup]="restoreForm"
+          novalidate>
+      <div class="modal-body">
+        <p>
+          To restore
+          <kbd>{{ poolName }}/{{ imageName }}@{{ imageId }}</kbd>, type the image's new name and click on
+          <kbd>Restore Image</kbd>.
+        </p>
+
+        <div class="form-group"
+             [ngClass]="{'has-error': restoreForm.showError('name', formDir)}">
+          <label for="name"
+                 i18n>New Name</label>
+          <input type="text"
+                 class="form-control"
+                 name="name"
+                 id="name"
+                 autocomplete="off"
+                 formControlName="name"
+                 placeholder="image_name"
```

Comment:
Is this the correct placeholder text? Don't forget i18n.

Reply:
Removed the placeholder.